schemachange/schemachange.c

```diff
@@ -334,9 +333,13 @@ int start_schema_change_tran(struct ireq *iq, tran_type *trans)
             rc = pthread_create(&tid, &gbl_pthread_attr_detached,
                                 (void *(*)(void *))do_schema_change_locked, s);
         } else {
-            Pthread_mutex_lock(&s->mtx);
             rc = pthread_create(&tid, &gbl_pthread_attr_detached,
                                 (void *(*)(void *))do_schema_change_tran, arg);
+            if (rc == 0) {
+                Pthread_mutex_lock(&s->mtxStart);
```

Comment:
Should lock mtxStart before creating thread.

Reply:
I don't understand.